src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,226 @@
+package duke;
+
+import duke.exception.DukeException;
+import duke.exception.emptyException;
+import duke.exception.nonMatchException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+import java.util.Scanner;
+//import java.util.ArrayList;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class Duke {
+
+    public static final int MAX_TASK_NUM = 100;
+    public static Task[] taskArray = new Task[MAX_TASK_NUM];
+    //ArrayList<Task> tasks = new ArrayList<>();
+
+    public static void printSeparator() {
+        String separator="____________________________________________________________";
+        System.out.println(separator);
+    }
+
+    public static int readFile(Task[] tasks){
+        try {
+            int sum=0;
+            File f = new File("/Users/wendyw/Desktop/study/cz2006(cs2113)/ip/data/duke.txt");
+            Scanner s = new Scanner(f);
+            while(s.hasNext()){
+                String line=s.nextLine();
+                boolean isDone=false;
+                String[] sentences=line.split(" \\| ");
+                if (sentences[0].equals("T")){
+                    tasks[sum]=new ToDo(sentences[2]);
+                    if (sentences[1].equals("1")){
+                        isDone=true;

```

Comment:
To enhance readability, perhaps add a space before and after the `=`. This also goes for all the `isDone` statements in this class.